src/api/generate-csv/map-application-to-csv/helpers/policy-type/index.ts

```diff
@@ -0,0 +1,31 @@
+import { FIELD_VALUES } from '../../../../constants';
+
+/**
+ * isSinglePolicyType
+ * Check if a policy type is 'Single contract policy'
+ * @param {String} Policy type
+ * @returns {Boolean}
+ */
+const isSinglePolicyType = (policyType: string) => {
+  if (policyType === FIELD_VALUES.POLICY_TYPE.SINGLE) {
```

Comment:
Ternary operator for reduced LOC.

Reply:
Ternary actually not required, made it even simpler 😎